app/editor/src/features/content/form/ContentStoryForm.tsx

```diff
@@ -41,6 +43,26 @@ export const ContentStoryForm: React.FC<IContentStoryFormProps> = ({
     initSummaryRequired ?? isSummaryRequired(values),
   );
 
+  const getTags = (e: string) => {
+    if (!checkTags) {
+      return;
+    }
+    const regex = new RegExp(/\[(.*?)\]/g);
```

Comment:
Is there any value to only parse the end of string (trim for empty spaces)?  The client will never add tags to the middle of a document.  This would also remove the risk of some random data coming in from 3rd parties.

Reply:
It would make my life easier ... hahaha
Probably there is a way, I'll just google it and make it work.
:)

Reply:
Found it ! And works good.